docs/tutorials/quickstart.md

```diff
@@ -0,0 +1,30 @@
+# Quickstart
+
+{cell=quickstart, display=false, output=false, results=false}
+```julia
+using Flux, Metalhead
+```
+
+Using a model from Metalhead is as simple as selecting a model from the table of [available models](#). For example, below we use the ResNet-50 model with pre-trained weights.
+{cell=quickstart}
+```julia
+using Flux, Metalhead
+
+model = resnet50(pretrain=true)
+```
+
+Now, we can use this model with Flux like any other model. Below, we train it on some randomly generated data.
+{cell=quickstart}
+```julia
+using Flux: onehotbatch
+using Statistics: mean
+
+batchsize = 8
+data = [(rand(Float32, 224, 224, 3, batchsize), onehotbatch(rand(1:1000), 1:1000))
+        for _ in 1:3]
+opt = ADAM()
+ps = Flux.params(model)
+loss(x, y, m) = Flux.Losses.logitcrossentropy(m(x), y)
+cb = () -> @show mean(loss(x, y, model) for (x, y) in data)
```

Comment:
the callback is called after each gradient step, so evaluating on the whole dataset it's not good even if this is an example with dummy data. Use manual loop instead?

Reply:
Yeah ideally, we would not do a training example but evaluate the pre-trained model on a sample from ImageNette in MLDatasets.jl. But to include that in the docs might mean we want to spend some time fusing FastAI.jl datasets and MLDatasets.jl better (ImageNette comes from FastAI.jl).

Reply:
I plan on doing a separate training loop example with a Jupyter notebook. But that requires https://github.com/MichaelHatherly/Publish.jl/issues/9. Doesn't seem like too hard a fix, but I just haven't had the time to look into it yet. The reason for preferring Jupyter notebooks is that training these models requires a GPU, so ideally, we would save the notebook (storing the output), and then we don't have to re-run the training to build the docs.